contrib/completion/zsh/_docker-compose

```diff
@@ -198,21 +198,23 @@ __docker-compose_subcommand() {
         (create)
             _arguments \
                 $opts_help \
-                "(--no-recreate --no-build)--force-recreate[Recreate containers even if their configuration and image haven't changed. Incompatible with --no-recreate.]" \
-                "(--force-recreate)--no-build[If containers already exist, don't recreate them. Incompatible with --force-recreate.]" \
-                "(--force-recreate)--no-recreate[Don't build an image, even if it's missing]" \
+                "(--no-recreate)--force-recreate[Recreate containers even if their configuration and image haven't changed. Incompatible with --no-recreate.]" \
```

Comment:
So as it turns out, `--build` is actually slightly different between `up` and `create`, so I've let that one be.

However, I've extracted `--force-recreate`, `--no-recreate`, `--no-build`, and all other duplicated flags into variables.